store/store.go

```diff
@@ -284,6 +286,8 @@ func init() {
 	if err != nil {
 		panic(err)
 	}
+	defaultConfig.DetailFields = jsonutil.StructFields((*snapDetails)(nil), "snap_yaml_raw")
+	defaultConfig.InfoFields = jsonutil.StructFields((*storeSnap)(nil))
```

Comment:
we should not ask for "snap-yaml" I think here as well, we don't expose it over the snapd API, it's useful (at least so far) only when we are about to install which is not the role of v2/snaps/info, also we will get one copy per channel which is quite a bit